src/sync/test.rs

```diff
@@ -1,7 +1,17 @@
+use pretty_assertions::assert_eq;
+
 use crate::{
     bson::{doc, Document},
     error::{CommandError, ErrorKind, Result},
-    options::{Acknowledgment, CollectionOptions, DatabaseOptions, FindOptions, WriteConcern},
+    options::{
+        Acknowledgment,
+        ClientOptions,
+        CollectionOptions,
+        DatabaseOptions,
+        FindOptions,
+        StreamAddress,
```

Comment:
The difficulty I had in parsing this to figure out that the difference was `StreamAddress` being added makes me wonder if it's worth updating our rustfmt config so that it always puts each item being imported on a separate line.

Reply:
I'm in favor, I always find that format much easier to read

Reply:
Any objections @patrickfreed?

Reply:
sgtm